### ISSUE TITLE
Social auth model in django

### ISSUE THREAD

**Issue by CodeMaxx** [Ferozepurwale/LendIt-App#5]
Create table for storing information extracted from facebook. Extract as much info as possible. 

Might require multiple tables.

**Comment next to each column in the table what it will be used for in the app**

Ping @nihal111 @arpan98 . Any comments?

**Comment by TheFlash98** [comment]
Useful link. Lists all that the FB login auth gives. May need some filtering to create a model. https://developers.facebook.com/docs/facebook-login/overview

**Comment by ydidwania** [comment]
Check the PR in Ferozepurwale/Lendit

**Comment by CodeMaxx** [comment]
@TheFlash98 @ydidwania Is the login being handled completely on the android side? I didn't see any entry for a token or anything on the server PR.

**Comment by TheFlash98** [comment]
Yes, the login happens only on the Android side. Do you want to shift that as well to the server? Right now you can press the `Login With Facebook` and your Facebook Id is displayed. Nothing goes to or comes from the server.

**Comment by CodeMaxx** [comment]
As long as we can support auto-login, doesn't matter where it is
